src/component.jl

```diff
@@ -390,6 +390,47 @@ function check_contiguous_forecasts(
     return first_initial_time, total_horizon
 end
 
+"""
+Efficiently add all forecasts in one component to another by copying the underlying
+references.
+
+# Arguments
+- `src::InfrastructureSystemsType`: Source component
+- `dst::InfrastructureSystemsType`: Destination component
+- `label_mapping::Dict = nothing`: Optionally map src labels to different dst labels.
+  If provided and src has a forecast with a label not present in label_mapping, that
+  forecast will not copied. If label_mapping is nothing then all forecasts will be copied
+  with src's labels.
+"""
+function copy_forecasts!(
```

Comment:
Why are these changes showing up?  They are in master.

Reply:
I don't know, I think that when I rebased they showed up here.